cpataltmetrik/webdriveriowithtypescript
# Added Bail out logic to framework

### Ticket Number / Feature:
Implementation of Bailout logic and bail report in the framework.
Added selenium standalone service to take care of browser updates.
### Description:
Added code in respective hooks and config file which will create the bail option.
### Test Execution Proof
![image](https://user-images.githubusercontent.com/51842349/167634998-426a99e7-808a-402b-9bb0-e495bf2a7164.png)
### File changes
package.json
config-wdio\wdio.local.conf.ts
### How to Execute Test Case:
```
1. Run the command npm run test-dev for local exeution.
2. Check the execution results with bail report.
```
